python/orca/src/bigdl/orca/learn/utils.py

```diff
@@ -291,6 +316,22 @@ def _dataframe_to_xshards(data, feature_cols, label_cols=None, accept_str_col=Fa
     return SparkXShards(shard_rdd)
 
 
+def _dataframe_to_xshards_of_pandas_df(data, feature_cols, label_cols=None, accept_str_col=False):
```

Comment:
Maybe we could make the function public

Reply:
Shall we also add some docs to explain the `data` as well as `feature_cols`/`label_cols` constraints?  e.g one feature_col corresponds to 1-d data